### PR TITLE
draftFrameworks created successfully

### PR DESCRIPTION
**Pull request by aman-tunerlabs**

### CODE REVIEW

File: controllers/v1/draftFrameworksController.js [hunk 1]
```diff
@@ -143,66 +96,97 @@ module.exports = class DraftFrameworks extends Abstract {
...
+*/
+
+  async list(req) {
+    return new Promise(async (resolve, reject) => {
+
+      try {
+
+        let frameworkDocument = await database.models.draftFrameworks.find({
+          userId: req.userDetails.id,
+          isDeleted: false
```
Comment (ajithtunerlabs) [inline controllers/v1/draftFrameworksController.js:117]: have you tested this isDeleted false is working or not, because we are not adding isDeleted false field when we create a document so it won't give you the expected result

File: module/draftFrameworks/validator/v1.js [hunk 2]
```diff
@@ -5,39 +5,11 @@ module.exports = (req) => {
             req.checkParams('_id').exists().withMessage("required framework id");
         },
         update: function () {
-            req.checkQuery('frameworkExternalId').exists().withMessage("required framework externalId");
```
Comment (ajithtunerlabs) [inline module/draftFrameworks/validator/v1.js:8]: why did you remove these validations?

File: controllers/v1/draftFrameworksController.js (hunk 1 above)
Comment (aman-tunerlabs) [reply controllers/v1/draftFrameworksController.js:117]: isDeleted field is in the draftFrameworks model. And by default it is false. Once we hit the delete api then only isDeleted is set to true. Yes i have checked it .

File: module/draftFrameworks/validator/v1.js (hunk 2 above)
Comment (aman-tunerlabs) [reply module/draftFrameworks/validator/v1.js:8]: This validations is no longer required.

Comment (aman-tunerlabs) [reply module/draftFrameworks/validator/v1.js:8]: I have added the commented validations .
